### ISSUE TITLE
@graphql-codegen/introspection isn’t installed by init wizard

### ISSUE THREAD

**Issue by dangodev** [dotansimha/graphql-code-generator#2004]
**Describe the bug**
Running `graphql-codegen init` on first startup will help you formulate your `codegen.yml` file. The last question: “Would you like to generate an introspection? (Y/n)” will add the introspection config. However, after that step’s complete, running `graphql-codegen` results in the following error:

```
graphql-codegen --config codegen.yml

  ✔ Parse configuration
  ❯ Generate outputs
  ✔ Parse configuration
  ❯ Generate outputs
    ✔ Generate src/types/graphql-global-types.ts
    ❯ Generate src/spec/graphql.schema.json
      ✔ Load GraphQL schemas
      ✔ Load GraphQL documents
      ✖ Generate
        → Unable to find template plugin matching introspection


 Found 1 error

  ✖ src/spec/graphql.schema.json
    Unable to find template plugin matching 'introspection'
    Install one of the following packages:


    - @graphql-codegen/introspection
    - @graphql-codegen/introspection-template
    - @graphql-codegen/introspection-plugin
    - graphql-codegen-introspection
    - graphql-codegen-introspection-template
    - graphql-codegen-introspection-plugin
    - codegen-introspection
    - codegen-introspection-template
    - introspection
```

For clarity: the other questions _did_ install packages to `package.json`, but introspection didn’t.

**To Reproduce**
Run `graphql-codegen init`, say “yes” for introspection.

1. My GraphQL schema:

N/A

2. My GraphQL operations:

N/A

3. My `codegen.yml` config file:

```yml
overwrite: true
schema: 'https://api.manifold.co/graphql'
documents: '/**/*.graphql'
generates:
  src/types/graphql-global-types.ts:
    plugins:
      - 'typescript'
      - 'typescript-operations'
  src/spec/graphql.schema.json:
    plugins:
      - 'introspection'

```

**Expected behavior**
Expected `init` to install all required packages.

**Environment:**

- OS: macOS
- `@graphql-codegen/cli@1.2.1`: 
- NodeJS: 12.4.0

**Additional context**
N/A

**Comment by dotansimha** [comment]
Thank you for reporting it, @dangodev ! 
Fixed in: https://github.com/dotansimha/graphql-code-generator/pull/2012

**Comment by dotansimha** [comment]
Fixed in 1.3.0 🎉
